### ISSUE TITLE
Installation on Raspbian doesn't work

### ISSUE THREAD

**Issue by christian-vorhemus** [Microsoft/onnxruntime#613]
**Describe the bug**
I'm following the steps described in the <a href="https://github.com/Microsoft/onnxruntime/blob/master/dockerfiles/Dockerfile.arm32v7">docker file</a> to get onnxruntime running on my Raspberry Pi 3, but the installation fails without a meaningful error description:

```
CMakeFiles/onnxruntime_providers.dir/build.make:959: recipe for target 'CMakeFiles/onnxruntime_providers.dir/home/pi/onnxruntime/onnxruntime/core/providers/cpu/tensor/onehot.cc.o' failed
make[2]: *** [CMakeFiles/onnxruntime_providers.dir/home/pi/onnxruntime/onnxruntime/core/providers/cpu/tensor/onehot.cc.o] Error 1
CMakeFiles/Makefile2:264: recipe for target 'CMakeFiles/onnxruntime_providers.dir/all' failed
make[1]: *** [CMakeFiles/onnxruntime_providers.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
Traceback (most recent call last):
  File "/home/pi/onnxruntime/tools/ci_build/build.py", line 683, in <module>
    sys.exit(main())
  File "/home/pi/onnxruntime/tools/ci_build/build.py", line 654, in main
    build_targets(cmake_path, build_dir, configs, args.parallel)
  File "/home/pi/onnxruntime/tools/ci_build/build.py", line 387, in build_targets
    run_subprocess(cmd_args)
  File "/home/pi/onnxruntime/tools/ci_build/build.py", line 162, in run_subprocess
    return subprocess.run(args, cwd=cwd, check=True, stdout=stdout, stderr=stderr, env=my_env, shell=shell)
  File "/usr/lib/python3.5/subprocess.py", line 398, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/usr/local/bin/cmake', '--build', '/home/pi/onnxruntime/build/Linux/Debug', '--config', 'Debug']' returned non-zero exit status 2
```

**System information**
- OS Platform and Distribution: Raspbian Stretch Lite
- ONNX Runtime installed from (source or binary): Source
- ONNX Runtime version: 0.3.0
- Python version: 3.5.3
- GCC/Compiler version: 6.3.0 20170516
- CUDA/cuDNN version: -
- CMake version: 3.12.3
- GPU model and memory: Raspberry Pi 3 Model B

**To Reproduce**
Follow the steps as described in the docker file and then run
`./build.sh --config Debug`
The last step I see before the error appears is
`[ 55%] Building CXX object CMakeFiles/onnxruntime_providers.dir/home/pi/onnxruntime/onnxruntime/core/providers/cpu/tensor/onehot.cc.o`

**Expected behavior**
The binaries should get built

**Comment by snnn** [comment]
Could you please redirect the error to a log file(e.g. build.log) and run:
```
  grep -C3 "error: " build.log
```

I guess it should be easy to fix.

**Comment by christian-vorhemus** [comment]
Thank you, as advised, I'll open a bug on Azure DevOps!

**Comment by snnn** [comment]
It's a compiler warning, we will fix it but the warning can be temporarily ignored. 
To ignore it:

Please open "tools/ci_build/build.py", search onnxruntime_DEV_MODE, remove the whole line. Then delete your build folder, run build.sh again.

Now I'll close this issue. @pranavsharma  will follow up on this. Thanks.

**Comment by ioarun** [comment]
Hi,
Thanks for the awesome library. I am facing the same issue on Raspberry Pi 4 (Raspbian - Buster). I followed the instructions given [here](https://github.com/microsoft/onnxruntime/blob/master/dockerfiles/README.md#arm-32v7) to build from scratch. Although the wheel file was generated, the same could not be installed on RPi.

Also, I had changed the first line in the docker file as : FROM balenalib/raspberrypi4-64-debian:buster-build 

I hope I am not missing something very obvious.

Thanks!
